### PR TITLE
Copy and show/hide password

### LINKED ISSUE #53: Extension item details: click to reveal and copy (login, password)
A quick search turned up some straight forward npm packages for both but not sure if they're worth incorporating as a dependency vs. rolling our own.

- click to reveal (input type password vs text)
  - https://www.npmjs.com/package/react-password-mask

- click to copy (grab input contents to clipboard)
  - https://www.npmjs.com/package/react-copy-to-clipboard

Any strong opinions one way or another (build ourselves vs package) @jimporter @linuxwolf?

### PR DESCRIPTION
**Pull request by jimporter**
This resolves #53. It's based on my other PRs, so just look at the last two commits.

### CODE REVIEW

File: src/webextension/locales/en-US/manage.ftl [hunk 1]
```diff
@@ -38,6 +38,7 @@ item-details-title = Title
 item-details-origin = Origin
 item-details-username = Username
```
Comment (linuxwolf) [inline src/webextension/locales/en-US/manage.ftl]: I'd really like the ability to copy the username, too.

Comment (devinreams) [reply src/webextension/locales/en-US/manage.ftl]: >I'd really like the ability to copy the username, too.

Great point 👍 I have some complex usernames and a shortcut here too would be nice.

**Comment by jimporter** [comment]
The actual failures are because Enzyme doesn't have some bits that `react-copy-to-clipboard` expects, so I'll have to shim it for now. A lot of this will probably be easier with Karma, but I'm waiting until after alpha for that, since it'd probably take a day or two to get everything working.

I just forgot to look at tests before I pushed this. vOv

**Comment by jimporter** [comment]
Actually, the failures were because the test code to figure out what button to click was very brittle. I fixed that, so it should be good for the foreseeable future now.

**Comment by jimporter** [comment]
I also added the ability to copy the username.

**Comment by jimporter** [comment]
Force-landing, since all I did was rebase.

**Pull request merged**